wikitemplate.md

```diff
@@ -67,6 +67,7 @@
 - [ ] Visit https://www.blizzard.com and then turn on script blocking, page should not load.
 - [ ] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/9/ when 3rd party cookies are blocked and not blank when 3rd party cookies are unblocked
 - [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run)
+- [ ] In `brave://settings/security`, choose a DNS provider from the providers listed under Use secure DNS, load https://browserleaks.com/dns, and verify your ISP's DNS resolvers aren't detected and shown; only your chosen DoH provider should appear.
```

Comment:
:+1: this should also have an item for `Open a Tor tab, load https://browserleaks.com/dns, and verify your ISP's DNS resolvers aren't detected and shown`

Reply:
Just added: https://github.com/brave/qa-resources/pull/236/commits/07602478a344efd02dc0d05295f89562eaef4ea4#diff-8e23795bde2c11770f8ae9394839337a39d31831496f063cc8d2e953569767dbR71